cms/admin/views.py

```diff
@@ -54,6 +66,10 @@ def revert_plugins(request, version_id, obj):
             if old.pk == plugin.pk:
                 plugin.save()
                 current_plugins.remove(old)
+
+    # TODO: the same rule for the next block (others) applies here, this should be wrapped in an atomic block
+    # see https://docs.djangoproject.com/en/1.9/topics/db/transactions/#controlling-transactions-explicitly
+    # block "Avoid catching exceptions inside atomic!"
```

Comment:
The exception is used here exactly to prevent the transaction to fail, and to provide an alternate way to recover the situation